appveyor.yml

```diff
@@ -73,6 +73,9 @@
       - nodejs_version: 11
         GYP_MSVS_VERSION: 2015
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - nodejs_version: 12
+        GYP_MSVS_VERSION: 2015
```

Comment:
Think this and 10/11 are actually supposed to be 2017 https://github.com/nodejs/build/blob/639b55d6e7efb911898c2ada97d0b4099ee86d4f/doc/windows-visualstudio-supported-versions.md#for-building-nodejs-core

Reply:
Updated.